[2022-08-02 17:44:05,394] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: trigger_rule_dag.task_1 scheduled__2022-08-01T00:00:00+00:00 [queued]>
[2022-08-02 17:44:05,408] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: trigger_rule_dag.task_1 scheduled__2022-08-01T00:00:00+00:00 [queued]>
[2022-08-02 17:44:05,409] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 17:44:05,410] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-08-02 17:44:05,410] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 17:44:05,442] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): task_1> on 2022-08-01 00:00:00+00:00
[2022-08-02 17:44:05,453] {standard_task_runner.py:52} INFO - Started process 17190 to run task
[2022-08-02 17:44:05,464] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'trigger_rule_dag', 'task_1', 'scheduled__2022-08-01T00:00:00+00:00', '--job-id', '283', '--raw', '--subdir', 'DAGS_FOLDER/trigger_rule_dag.py', '--cfg-path', '/tmp/tmp1te1vgqi', '--error-file', '/tmp/tmpun_fc_sl']
[2022-08-02 17:44:05,471] {standard_task_runner.py:80} INFO - Job 283: Subtask task_1
[2022-08-02 17:44:05,625] {task_command.py:369} INFO - Running <TaskInstance: trigger_rule_dag.task_1 scheduled__2022-08-01T00:00:00+00:00 [running]> on host 2525a2c6e2b8
[2022-08-02 17:44:05,862] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=trigger_rule_dag
AIRFLOW_CTX_TASK_ID=task_1
AIRFLOW_CTX_EXECUTION_DATE=2022-08-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-01T00:00:00+00:00
[2022-08-02 17:44:05,865] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-02 17:44:05,869] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 30']
[2022-08-02 17:44:05,891] {subprocess.py:85} INFO - Output:
[2022-08-02 17:44:36,758] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-02 17:44:36,776] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=trigger_rule_dag, task_id=task_1, execution_date=20220801T000000, start_date=20220802T174405, end_date=20220802T174436
[2022-08-02 17:44:36,801] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-02 17:44:36,827] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
